presto-hive/src/test/java/io/prestosql/plugin/hive/TestBackgroundHiveSplitLoader.java

```diff
@@ -566,6 +571,41 @@ public void testHive2VersionedFullAcidTableFails()
         deleteRecursively(tablePath, ALLOW_INSECURE);
     }
 
+    @Test
+    public void testValidateFileBuckets()
+    {
+        ListMultimap<Integer, LocatedFileStatus> bucketFiles = ArrayListMultimap.create();
+        bucketFiles.put(1, null);
+        bucketFiles.put(3, null);
+        bucketFiles.put(4, null);
+        bucketFiles.put(6, null);
+        bucketFiles.put(9, null);
+
+        assertThatThrownBy(() -> BackgroundHiveSplitLoader.validateFileBuckets(bucketFiles, 1, "tableName", "partitionName"))
```

Comment:
Static import `validateFileBuckets` for readability

Reply:
You can do this using
```java
assertPrestoExceptionThrownBy(() -> validateFileBuckets(bucketFiles, 1, "tableName", "partitionName"))
        .hasErrorCode(HIVE_INVALID_BUCKET_FILES)
        .hasMessage(...);

Reply:
got it